.claude/skills/run-ci-tests/SKILL.md

```
@@ -0,0 +1,169 @@
+allowed-tools: Read, Grep, Glob, Bash, AskUserQuestion, TodoWrite
+---
+
+# Run CI Tests Locally
+
+Reproduce the `release-pipeline.yml` test jobs on a developer machine, AS IS: the same
+scripts, the same flags, the same per-job environment. Do not "fix" or rewrite the CI
+scripts; adapt the environment around them and disclose every adaptation in the report.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)